docs/content/integrations/bigquery/reference.mdx

```diff
@@ -517,6 +517,16 @@ defs = Definitions(
 </TabItem>
 </TabGroup>
 
+<Note>
+  Note: In order to load data from BigQuery as a PySpark DataFrame, the BigQuery
```

Comment:
```suggestion
  <strong>Note:</strong> In order to load data from BigQuery as a PySpark DataFrame, the BigQuery
```

This is the style I've been having us use for this type of sentence structure - specifically, `Note:` is bolded and the rest of the sentence isn't. Just makes it stand out a bit more.